src/app/EditProfileUseCaseFactory.java

```diff
@@ -150,4 +131,11 @@ private static EditSpotifyHandleController createEditSpotifyHandleUseCase(ViewMa
 
         return new EditSpotifyHandleController(editSpotifyHandleInteractor);
     }
+
+    private static GoBackController createGoBackUseCase(ViewManagerModel viewManagerModel, GoBackViewModel goBackViewModel) {
```

Comment:
Just to clarify, which view will the go back button be located? Does it allow you to go back to every previous view, or just the logged in view?

Reply:
It depends on the specific implementation of the go-back button, but I think just going back to the logged-in view is more doable for now.